frameaccurate/js/FrameAccurateControls.ts

```diff
@@ -0,0 +1,255 @@
+
+class FrameAccurateControls {
+
+  constructor(private player: any, private assetDescription: AssetDescription) {
```

Comment:
I'd prefer putting the variable explicitly in the class (even though it technically doesn't make a difference)